README.md

```diff
@@ -23,6 +25,13 @@ If you run into build issues, clean the Sphinx output directory and AutoAPI outp
 cd docs && make clean
 ```
 
+When writing documentation you can use the convenience command below. This rebuilds 
```

Comment:
@Miauwkeru I know you played around a fair bit with the build stuff for Sphinx, do you have an opinion on these additions?

Reply:
When used correctly `sphinx-autobuild` should not "keep writing stuff". I encountered this too, you need to have built the project manually at least once (using `make html`) before using the `make watch` command. 

The behavior afterwards should be that only files you change will be re-built by sphinx.

Reply:
Added in [3417461](https://github.com/fox-it/dissect-docs/pull/36/commits/3417461df1dfa2e39d04fee6494d0484ab5fe177)